### PR TITLE
Add option to disable kube-health-monitor

### PR DESCRIPTION
**Pull request by meysholdt**
Fixes https://github.com/gitpod-io/gitpod/issues/5265

Add an option to our helm chart to disable kube-health-monitor. 

Dev-Loop:
* run `docker run --rm --privileged --pid=host  ubuntu /usr/bin/nsenter -t "1" -a bash -c "$(cat stop.sh)"` on a node with `stop.sh` being the bash snippet from this PR.

Testing
* Look at the [logs on core-dev](https://console.cloud.google.com/logs/query;query=%22disable-kube-health-monitor%22;timeRange=PT15M;cursorTimestamp=2021-08-19T10:53:20Z?project=gitpod-core-dev)

- [x] /werft with-clean-slate-deployment

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **fntlnz** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @fntlnz` in a comment when ready.

Associated issue: *#5265*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by wulfthimm — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: e834dd13a1f97629437eb5957a497951b015edce</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5267#pullrequestreview-733853875" title="Approved">wulfthimm</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **fntlnz** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @fntlnz` in a comment when ready.

Associated issue: *#5265*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by meysholdt** [comment]
Why does @roboquat try to summon Lorenzo?

**Review by geropl — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5267#pullrequestreview-733861250" title="Approved">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5267#pullrequestreview-733853875" title="Approved">wulfthimm</a>*

Associated issue: *#5265*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [geropl]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
